rtl/include/eclhelper.hpp

```diff
@@ -473,6 +475,7 @@ interface IOutputMetaData : public IRecordSize
     virtual void process(const byte * self, IFieldProcessor & target, unsigned from, unsigned to) {}            // from and to are *hints* for the range of fields to call through with
     virtual void walkIndirectMembers(const byte * self, IIndirectMemberVisitor & visitor) = 0;
     virtual IOutputMetaData * queryChildMeta(unsigned i) = 0;
+    virtual const RtlRecord *queryRecordAccessor(bool expand=true) const { return NULL; }
```

Comment:
possibly better without a default parameter so it is clear which one is being used.